app/src/pages/inside/common/launchFiltersSection/launchFiltersSection.jsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+
+import { FilterEntitiesContainer } from 'components/filterEntities/containers';
+import { LaunchFiltersContainer } from 'pages/inside/common/launchFiltersContainer/launchFiltersContainer';
+import { LaunchFiltersToolbar } from 'pages/inside/common/launchFiltersToolbar';
+
+export const LaunchFiltersSection = ({ level }) => (
+  <LaunchFiltersContainer
+    render={({
+      launchFilters,
+      activeFilterId,
+      activeFilter,
+      activeFilterEntities,
+      onSelectFilter,
+      onRemoveFilter,
+      onChangeFilter,
+      onResetFilter,
+    }) => (
+      <FilterEntitiesContainer
+        level={level}
```

Comment:
This section contains only launch level filters. No need to receive it from props.
`level={LEVEL_LAUNCH}`

Reply:
Removed